build.gradle

```diff
@@ -20,7 +20,7 @@ plugins {
 apply plugin: 'witness'
 apply plugin: 'maven'
 
-def app_version = '0.10.1'
+def app_version = '0.12.0'
```

Comment:
Can you remove the version change? If the next release includes both this PR and #71, then of course the release version will be 0.11.

In my experience it's easier to only increment the version number in a release commit, not for every PR that changes something.